launcher/sdw_updater_gui/Updater.py

```diff
@@ -0,0 +1,350 @@
+#!/usr/bin/env python3
+"""
+Utility library for checking and applying SecureDrop Workstation VM updates.
+
+This library is meant to be called by the SecureDrop launcher, which
+is opened by the user when clicking on the desktop, opening sdw-laucher.py
+from the parent directory.
+"""
+
+import logging
+import os
+import subprocess
+from datetime import datetime
+from enum import Enum
+
+DEFAULT_HOME = os.path.join(os.path.expanduser("~"), ".securedrop_launcher")
+
+FLAG_FILE_STATUS_SD_SVS = "/home/user/sdw-update-flag"
+FLAG_FILE_LAST_UPDATED_SD_SVS = "/home/user/sdw-last-updated"
+FLAG_FILE_LAST_UPDATED_DOM0 = os.path.join(DEFAULT_HOME, "sdw-last-updated")
```

Comment:
one other thought: can we also store the content of `FLAG_FILE_STATUS_SD_SVS` in dom0 somewhere? The reason is that we need from dom0 to read `/home/user/sdw-update-flag` in `sd-svs` (which might be shutdown) to distinguish when the user hasn't updated their workstation for 5+ days but just because they just started their workstation after a 5 day period ([case B here](https://docs.google.com/document/d/1cCpcpLRiKC0tm_vsD4k4Q1Fr5_R34htCb8FNTV8mRCI/edit)).